app/models/comment.rb

```diff
@@ -99,6 +99,9 @@ def self.federated_class
     def initialize(person, target, text)
       @text = text
       super(person, target)
+      authors = target.comments_authors
+      author_people = Person.find(authors)
+      @dispatcher_opts = {:additional_subscribers => author_people}
```

Comment:
I don't like too much that where super is called is important. How about having `@dispatcher_opts ||= {}` in the parent?